notify/twilio.py

```diff
@@ -0,0 +1,84 @@
+import json
+import logging
+import os
+
+from twilio.rest import Client
+
+from .utils import shorten_url
+
+states = json.loads(os.environ["STATES"])
+
+def format_available_message(locations):
+    message = "Vaccine appointments available at {} location{}:".format(
+        "these" if len(locations) > 1 else "this",
+        "s" if len(locations) > 1 else "",
+    )
+    for location in locations:
+        if "earliest_appointment_day" in location:
+            if (
+                location["earliest_appointment_day"]
+                == location["latest_appointment_day"]
+            ):
+                day_string = " on {}".format(location["earliest_appointment_day"])
+            else:
+                day_string = " from {} to {}".format(
+                    location["earliest_appointment_day"],
+                    location["latest_appointment_day"],
+                )
+        else:
+            day_string = ""
+
+        message += "\n• {}{}{}. Sign up here: {}{}".format(
+            "{}: ".format(location["state"])
+            if (len(states) > 1 and "state" in location)
+            else "",
+            location["name"],
+            day_string,
+            shorten_url(location["link"]),
+            ", zip code {}".format(location["zip"]) if "zip" in location else "",
+        )
+    return message
+
+
+def format_unavailable_message(locations):
+    message = "Vaccine appointments no longer available at {} location{}:".format(
+        "these" if len(locations) > 1 else "this",
+        "s" if len(locations) > 1 else "",
+    )
+    for location in locations:
+        message += "\n• {}{}".format(
+            location["state"]
+            if (len(states) > 1 and "state" in location)
+            else "",
+            location["name"],
+        )
+    return message
+
+
+def send_message_to_twilio(message):
+    account_sid = os.environ['TWILIO_ACCOUNT_SID']
+    auth_token = os.environ['TWILIO_AUTH_TOKEN']
+    client = Client(account_sid, auth_token)
+
+    if "TWILIO_TO_NUMBERS" in os.environ:
+        recipients = json.loads(os.environ["TWILIO_TO_NUMBERS"])
```

Comment:
Thoughts on making this comma-separated instead of json?
I should have done that on `os.environ['STATES']`, but I don't really want to make a breaking change now that people are using that. I'm torn between making this consistent with states, or doing it the simpler way.

Reply:
I don't have a strong opinion either way. There may be some clarity with the JSON version that it's explicitly an array of strings for the phone numbers

Reply:
Either way is fine with me.

Reply:
OK, I think I'll keep it as is